clean-sensitive-data/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "lakshanweerasinghe"
```

Comment:
Shall we change the organization name to something like `ballerina` or similar one. WDYT?

Reply:
IMO it's better to have some other name than `ballerina`, since someone with access to `ballerina` org can accidentally push this repo.